private/buf/cmd/buf/buf.go

```diff
@@ -289,56 +274,29 @@ func NewRootCommand(name string) *appcmd.Command {
 							},
 						},
 					},
+					appcmd.NewDeletedCommand("push", betaPushDeprecationMessage),
 				},
 			},
 			{
-				Use:   "experimental",
-				Short: "Experimental commands. Unstable and will likely change.",
-				Deprecated: `use "beta" instead.
-We recommend migrating, however this command continues to work.
-See https://docs.buf.build/faq for more details.`,
-				Hidden: true,
+				Use:        "experimental",
+				Short:      "Experimental commands. Unstable and will likely change.",
+				Deprecated: experimentalDeprecationMessage,
+				Hidden:     true,
 				SubCommands: []*appcmd.Command{
 					{
 						Use:        "image",
 						Short:      "Work with Images and FileDescriptorSets.",
 						Deprecated: imageDeprecationMessage,
 						Hidden:     true,
 						SubCommands: []*appcmd.Command{
-							convert.NewCommand(
-								"convert",
-								builder,
-								imageDeprecationMessage,
-								true,
-							),
-						},
-					},
-				},
-			},
-			{
-				Use:    "alpha",
-				Short:  "Alpha commands. These are so early in development that they should not be used except in development.",
-				Hidden: true,
-				SubCommands: []*appcmd.Command{
-					{
-						Use:   "registry",
-						Short: "Interact with the Buf Schema Registry.",
-						SubCommands: []*appcmd.Command{
-							{
-								Use:   "token",
-								Short: "Token commands.",
-								SubCommands: []*appcmd.Command{
-									tokencreate.NewCommand("create", builder),
-									tokenget.NewCommand("get", builder),
-									tokenlist.NewCommand("list", builder),
-									tokendelete.NewCommand("delete", builder),
-								},
-							},
```

Comment:
I figure we may as well clean these up along the way - we don't have any plans for them in the near term, and we can keep each of the command implementations around if we want, but these aren't providing any value as-is. I'm happy to revert this if we feel strongly about keeping them in.